analysis/analysis.py

```diff
@@ -43,6 +43,77 @@ def compute_total_view_time(student_data):
     return total_view_time
 
 
+def get_word_frequency_differences(student_data):
+    """
+    Looks over the data and compares responses from people who have read the text vs.
+    people who have not read the text before this exercise
+    :return: a list of word frequency differences, by increasing order of frequency differences
+    """
+
+    # Iterate through all data, and separate ids of students who have vs. have not read the text
+    yes_id = []
+    no_id = []
+
+    for response in student_data:
+        if (response['question'].find('Have you encountered this text before') == 0
```

Comment:
rather than calling ```find()``` on these strings and doing your if on the return value, this pattern would be clearer:
```
if 'Have you encountered this text before' in response['question']:
```
(no magic numbers -- having to remember that the return value of ```str.find()``` is the index of the found string... that -1 means it's not there, etc.)